Source/iop/Iop_Dmacman.cpp

```diff
@@ -9,9 +9,15 @@ using namespace Iop;
 
 #define FUNCTIONID_DMACREQUEST 28
 #define FUNCTIONID_DMACTRANSFER 32
+#define FUNCTIONID_DMACSETVAL 33
```

Comment:
For the function names here, we should stay close to what is in PS2SDK: https://github.com/ps2dev/ps2sdk/blob/8b7579979db87ace4b0aa5693a8a560d15224a96/iop/kernel/include/dmacman.h#L143.

So, it should be:
`DmacChSetDpcr`
`DmacEnable`
`DmacDisable`